cumulusci/core/runtime.py

```diff
@@ -53,6 +55,14 @@ def keychain_cls(self):
     def get_keychain_class(self):
         return None
 
+    @property
+    def callback_cls(self):
+        klass = self.get_callback_class()
+        return klass or self.callback_class
+
+    def get_callback_class(self):
+        return None
```

Comment:
Do we have an existing use case where we need this callable rather than just using self.callback_class?

Reply:
I suspect preflight and real-installation-jobs will need different callbacks in MetaDeploy. In metadeploy at present, they're two sibling subclasses of BaseFlow in order to do their thing.